### PR TITLE
USHIFT-2125: Check devices' partitions for lvm

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-ocp-conformance-rhel-eus periodic-ci-openshift-microshift-main-clusterbot periodic-ci-openshift-microshift-release-4.15-clusterbot

File: ci-operator/step-registry/openshift/microshift/infra/lvm-install/openshift-microshift-infra-lvm-install-commands.sh [hunk 1]
```diff
@@ -21,7 +21,14 @@ chmod 0600 "${HOME}/.ssh/config"
...
 if [[ "${EC2_INSTANCE_TYPE%.*}" =~ .*"g".* || "${EC2_INSTANCE_TYPE%.*}" =~ "t3".* ]]; then
-  device="/dev/nvme1n1"
+  # Sometimes, devices are in different order and nvme1 stores OS while nvme0 should hold LVM for topolvm.
+  # If /dev/nvme0 is already partitioned (operating system), then use nvme1 for lvm.
+  # If `partx /dev/nvme0n1` fails (rc=1), it couldn't read partition table, so it's the one to use for lvm.
+  if ssh "${INSTANCE_PREFIX}" "sudo partx /dev/nvme0n1"; then
+    device="/dev/nvme1n1"
+  else
+    device="/dev/nvme0n1"
+  fi
```
Comment (ggiguash) [inline ci-operator/step-registry/openshift/microshift/infra/lvm-install/openshift-microshift-infra-lvm-install-commands.sh:31]: Nice! Simple and elegant solution...

**Pull request merged**